android/expoview/src/main/java/host/exp/exponent/experience/ReactNativeActivity.java

```diff
@@ -220,9 +225,20 @@ public void run() {
 
   @Override
   public boolean onKeyUp(int keyCode, KeyEvent event) {
-    if (keyCode == KeyEvent.KEYCODE_MENU && mReactInstanceManager != null && mReactInstanceManager.isNotNull() && !mIsCrashed) {
-      mReactInstanceManager.call("showDevOptionsDialog");
-      return true;
+    if (mReactInstanceManager != null && mReactInstanceManager.isNotNull() && !mIsCrashed) {
+      if (keyCode == KeyEvent.KEYCODE_MENU) {
+        mReactInstanceManager.call("showDevOptionsDialog");
+        return true;
+      }
+      DevSupportManager devSupportManager = (DevSupportManager) mReactInstanceManager.call("getDevSupportManager");
```

Comment:
we have to keep things as `RNObject`s here, this won't work for versioned classes. If you do `callRecursive` instead of `call` you'll get an `RNObject` back and then you can call `.call("getDevSupportEnabled")` on that.